src/_flow/tracer.js

```diff
@@ -20,4 +20,5 @@ declare type startSpanOptions = {
   references?: Array<Reference>,
   tags?: any,
   startTime?: number,
+  traceId128bit?: boolean,
```

Comment:
why is this needed? `startSpan()` is an OpenTracing API function, we cannot change its signature in Jaeger. This flag is only needed as tracer-level flag, not span-level.

Reply:
didn't know that, I'll move it